.drone.star

```diff
@@ -51,7 +51,6 @@ config = {
         "multipleVersions": {
             "phpVersions": [
                 DEFAULT_PHP_VERSION,
-                "7.3",
```

Comment:
Actually you can remove the whole thing, and just say:
```
"phan": True,
```

Because it defaults to running just for `DEFAULT_PHP_VERSION`